test/Market.t.sol

```diff
@@ -265,37 +264,22 @@ contract Market_replace_reverts_ReplacementAmountExceedsNrtDeficit is
   function setUp() external {
     _listRemovals();
     _createCertificate();
-
     _market.grantRole({
       role: _market.MARKET_ADMIN_ROLE(),
       account: _namedAccounts.admin
     });
-    _removal.grantRole({
-      role: _removal.RELEASER_ROLE(),
-      account: _namedAccounts.admin
-    });
-    uint256 amount = _market.calculateCheckoutTotal(_amountToReplace);
-    vm.startPrank(_namedAccounts.admin);
-    assertEq(_certificate.getNrtDeficit(), 0); // no discrepancy
-    _bpNori.deposit(_namedAccounts.admin, abi.encode(amount));
-    _bpNori.approve(address(_market), amount);
```

Comment:
removed several of these blocks of code as they were not needed in all test contexts